scip-java/src/main/scala/com/sourcegraph/scip_java/buildtools/BazelBuildTool.scala

```diff
@@ -12,16 +25,225 @@ class BazelBuildTool(index: IndexCommand) extends BuildTool("Bazel", index) {
+    val commandResult = index
       .app
-      .reporter
-      .error(
-        "The `scip-java index` command does not work with Bazel builds because Bazel " +
-          "requires custom configuration to be added to WORKSPACE and BUILD files. " +
-          "The scip-java repository contains an example WORKSPACE file to demonstrate " +
-          "how to use scip-java with Bazel https://github.com/sourcegraph/scip-java/blob/main/examples/bazel-example/WORKSPACE"
+      .process(buildCommand)
+      .call(check = false, stderr = Readlines(sandbox))
+    if (commandResult.exitCode != 0) {
+      if (index.bazelAutorunSandboxCommand && sandbox.commandLines().nonEmpty) {
+        index
+          .app
+          .info(
+            "Automatically re-running sandbox command to help debug the problem."
+          )
+        index
+          .app
+          .process("bash", "-c", sandbox.commandLines().mkString("\n"))
+          .call(check = false, stdout = os.Inherit, stderr = os.Inherit)
+      }
+      index
+        .app
+        .error(
+          s"""To reproduce the failed Bazel command without scip-java, run the following command:
+             |
+             |  bazel build ${targetSpecs.mkString(" ")}
+             |
+             |To narrow the set of targets to index or pass additional flags to Bazel, include extra arguments index after -- like below:
+             |
+             |  scip-java index --bazel-scip-java-binary=... -- //custom/target --sandbox_debug
+             |""".stripMargin
+        )
+      commandResult.exitCode
+    } else {
+      0
+    }
+  }
+
+  private def aggregateScipFiles(): Unit = {
+    // Final step after running the aspect: aggregate all the generated `*.scip` files into a single index.scip file.
+    // We have to do this step outside of Bazel because Bazel does not allow actions to generate outputs outside
+    // of the bazel-out directory. Ideally, we should be able to implement the aggregation step inside Bazel
+    // and only copy the resulting index.scip file into the root of the workspace. However, to begin with, we
+    // walk the bazel-bin/ directory to concatenate the `*.scip` files even if this is not the idiomatic way to
+    // do it (in general, scanning the bazel-bin/ directory is a big no no).
+    Files.deleteIfExists(index.finalOutput)
+    Files.createDirectories(index.finalOutput.getParent)
+    val scipPattern = FileSystems.getDefault.getPathMatcher("glob:**.scip")
+    val bazelOut = index.workingDirectory.resolve("bazel-out")
+    if (!Files.exists(bazelOut)) {
+      index
+        .app
+        .error(
+          s"doing nothing, the directory $bazelOut does not exist. " +
+            s"The most likely cause for this problem is that there are no Java targets in this Bazel workspace. " +
+            s"Please report an issue to the scip-java issue tracker if the command `bazel query 'kind(java_*, //...)'` returns non-empty output."
```

Comment:
_nit:_ don't need interpolators on the last two strings, could make this a stripMargin thing

Reply:
Fixed.